daemon/emer-daemon.c

```diff
@@ -1665,3 +1875,63 @@ emer_daemon_get_permissions_provider (EmerDaemon *self)
   EmerDaemonPrivate *priv = emer_daemon_get_instance_private (self);
   return priv->permissions_provider;
 }
+
+void
+emer_daemon_start_aggregate_timer (EmerDaemon            *self,
+                                   GDBusMethodInvocation *invocation,
```

Comment:
Rather than passing the invocation in here, can you make this return the timer on success and set a `GError **` on failure? There's no other DBus logic in this source file.